Makefile

```diff
@@ -17,15 +35,58 @@ shortcheck: build
 bench: build
 	go test -bench . ./...
 
+clean:
+	go clean -i ./...
+
+deps:
+	go get golang.org/x/sys/unix
+	go get golang.org/x/crypto/ssh/terminal
+	go get -t github.com/zalando/skipper/...
+	./etcd/install.sh
+	go get github.com/tools/godep
+	godep restore ./...
+
 vet: $(SOURCES)
 	go vet ./...
 
-fmt:
+fmt: $(SOURCES)
 	gofmt -w $(SOURCES)
 
-checkfmt:
-	if [[ $$(gofmt -d $(SOURCES)) ]]; then false; else true; fi
+check-fmt: $(SOURCES)
+	if [ "$$(gofmt -d $(SOURCES))" != "" ]; then false; else true; fi
```

Comment:
couldn't this be achieved with the `test` command? An if/else that does nothing but return true/false always looks weird to me